<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>648</width>
    <height>613</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>621</width>
      <height>551</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>5</number>
    </property>
    <widget class="QWidget" name="startPage">
     <widget class="QSpinBox" name="spinBoxMaxScores">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>220</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>180</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Укажите максимальное количество баллов</string>
      </property>
     </widget>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>500</y>
        <width>156</width>
        <height>23</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Ok</set>
      </property>
      <property name="centerButtons">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_list">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>290</y>
        <width>171</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Общий список</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Две группы раздельно</string>
       </property>
      </item>
     </widget>
     <widget class="QTextEdit" name="textInfo">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>260</y>
        <width>291</width>
        <height>151</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>551</width>
        <height>179</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="button_Mann">
         <property name="text">
          <string>Метод Манна-Уитни</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_Pirson">
         <property name="text">
          <string>Метод Пирсона</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_K_S">
         <property name="text">
          <string>Метод Колмогорова-Смирнова</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="mannPage">
     <widget class="QTableWidget" name="table_input">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>391</width>
        <height>391</height>
       </rect>
      </property>
     </widget>
     <widget class="QDialogButtonBox" name="buttonBox_2">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>510</y>
        <width>156</width>
        <height>23</height>
       </rect>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonChangeName">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>420</y>
        <width>191</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Изменить названия групп</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_3">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>460</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_4">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>460</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="button_calc">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>230</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>РАССЧЁТ</string>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>426</y>
        <width>257</width>
        <height>81</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>Удалить выделенные</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Добавить строку</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5">
     <widget class="QTableWidget" name="table_output">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>181</width>
        <height>421</height>
       </rect>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>60</y>
        <width>351</width>
        <height>261</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_results">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>350</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>110</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Old English Text MT</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Группа 1</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>230</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Old English Text MT</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Группа 2</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>160</y>
        <width>281</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Reference Sans Serif</family>
        <pointsize>12</pointsize>
        <italic>true</italic>
       </font>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_2">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>290</y>
        <width>281</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Reference Sans Serif</family>
        <pointsize>12</pointsize>
        <italic>true</italic>
       </font>
      </property>
     </widget>
     <widget class="QPushButton" name="calculateButton">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>420</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Poor Richard</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Рассчитать</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="mannPage2">
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>541</width>
        <height>421</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>450</y>
        <width>461</width>
        <height>80</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="calculateButton2">
         <property name="text">
          <string>Рассчитать</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox">
         <property name="font">
          <font>
           <family>Ravie</family>
           <weight>75</weight>
           <italic>true</italic>
           <bold>true</bold>
          </font>
         </property>
         <item>
          <property name="text">
           <string>Контрольная группа</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Экспериментальная группа</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="addButton">
         <property name="text">
          <string>Добавить</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="pirson_kolmPage">
     <widget class="QTableWidget" name="table_pirson">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>241</width>
        <height>371</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelNamePirson">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>10</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Метод Пирсона</string>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView_2">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>60</y>
        <width>291</width>
        <height>221</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelPirsonResults">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>310</y>
        <width>281</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>440</y>
        <width>366</width>
        <height>80</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="button_pirson_addrow">
         <property name="text">
          <string>Добавить строку</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_pirson_deleterow">
         <property name="text">
          <string>Удалить строку</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_pirson_calc">
         <property name="text">
          <string>Рассчёт</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>648</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="actions">
    <property name="title">
     <string>Операции</string>
    </property>
    <addaction name="action_csv"/>
    <addaction name="action_input"/>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="help">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="action_help"/>
   </widget>
   <addaction name="actions"/>
   <addaction name="help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_csv">
   <property name="text">
    <string>Считать из .csv</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="action_input">
   <property name="text">
    <string>Ввести вручную</string>
   </property>
  </action>
  <action name="action_help">
   <property name="text">
    <string>Помощь</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
